t/04i18n.t

```diff
@@ -1,9 +1,12 @@
 #!perl -w
-
 use strict;
-use Act::Config;
+use warnings;
 
 use Test::More;
+use Test::Lib;
+
+use Act::Config;
+Act::Config->load_configs('t');
```

Comment:
The repository does not have configuration files under t/.  If they are in your working copy and you intended to push them, please review .gitignore which excludes conf and actdocs.

Reply:
No, this is a remnant of custom configuration files.

Reply:
And I removed it